clipper_admin/clipper_admin/nomad/consul_dns.py

```diff
@@ -0,0 +1,18 @@
+from .dns import DNS
+import dns.resolver
+import socket
+
+class ConsulDNS(DNS):
```

Comment:
Couldn't find the usage in this PR. Where is it used?

Reply:
It should be used on the client side as follows:

```
from clipper_admin.deployers import python as python_deployer
from clipper_admin import ClipperConnection, DockerContainerManager, NomadContainerManager, ConsulDNS

nomad_ip_addr = '10.65.30.43'
    dns = ConsulDNS() # We use Consul for DNS resolution
    container_manager = NomadContainerManager(
        nomad_ip=nomad_ip_addr,
        dns=dns
    )
    clipper_conn = ClipperConnection(container_manager)
    clipper_conn.connect()
```

I will document this as well